mappings/net/minecraft/world/gen/feature/FortressFeature.mapping

```diff
@@ -0,0 +1,10 @@
+CLASS net/minecraft/unmapped/C_cncrqcde net/minecraft/world/gen/feature/FortressFeature
```

Comment:
Is this specific to nether fortresses or is this more of an abstract fortress feature? If it's a nether fortress feature, make it `NetherFortressFeature`, since that's in line with `NetherFortressGenerator`

Reply:
Renamed it